examples/example_02_electrolyzer/example_run.ipynb

```diff
@@ -12,7 +12,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
```

Comment:
we probably don't want to commit this, I'd recommend re-running the entire notebook and committing, or removing this change